slave_poller.go

```diff
@@ -101,7 +101,7 @@ func retrieveStats(c *http.Client, stats *[]MonitoredTask, url string) error {
 }
 
 // Periodically queries a Mesos slave and updates statistics of each running task
-func slavePoller(c *http.Client, conf *Config, frameworkRegistry *frameworkRegistry, slave Slave) {
+func slavePoller(c *http.Client, conf *Config, frameworkRegistry *frameworkRegistry, slave Slave, erroredSlaves *map[string]struct{}) {
```

Comment:
In Go, maps [are references](https://golang.org/doc/effective_go.html#maps) already. You can leave out the pointer.